fe/fe-core/src/main/java/com/starrocks/transaction/OlapTableTxnStateListener.java

```diff
@@ -277,7 +278,7 @@ public void postAbort(TransactionState txnState, List<TabletFailInfo> failedTabl
                     }
                 }
             } catch (Exception e) {
-                LOG.warn(e);
+                LOG.warn(DebugUtil.getStackTrace(e));
```

Comment:
```suggestion
                LOG.warn("Fail to execute postAbort", e);
```
In this way, the stack of `e` will be printed